packages/@react-aria/autocomplete/src/useSearchAutocomplete.ts

```diff
@@ -55,7 +55,9 @@ export function useSearchAutocomplete<T>(props: AriaSearchAutocompleteOptions<T>
     listBoxRef,
     keyboardDelegate,
     onSubmit = () => {},
-    onClear
+    onClear,
+    onKeyDown,
+    onKeyUp
```

Comment:
do we need to worry about duplicates for any other events?
onInputChange
onFocusChange
onOpenChange

Reply:
We do need to fix `onFocusChange`. The other two are not used by useSearchField, so those are clear.